src/tiled/mapdocumentactionhandler.cpp

```diff
@@ -706,6 +713,19 @@ void MapDocumentActionHandler::selectNextLayer()
         mMapDocument->switchSelectedLayers({ nextLayer });
 }
 
+void MapDocumentActionHandler::selectAllLayers()
+{
+    if (!mMapDocument)
+        return;
+    QList<Layer *> layersToSelect;
+
+    for (Layer *layer : mMapDocument->map()->allLayers()) {
+        layersToSelect.append(layer);
+    }
```

Comment:
For single-line bodies, please leave out the braces.

Reply:
Will do